discovery-infra/test_infra/assisted_service_api.py

```diff
@@ -306,19 +306,19 @@ def get_events(self, cluster_id: str, host_id: Optional[str] = "") -> dict:
 
         return json.loads(response.data)
 
-    def get_all_events(self, cluster_id: str, host_id: Optional[str] = "") -> dict:
```

Comment:
No need to `get_all_events` anymore. Use only `get_events` with provided `categories` as a parameter